docs/administrator-docs/installing.md

```diff
@@ -20,11 +20,23 @@ The Jellyfin Docker image is available on [Docker Hub](https://hub.docker.com/r/
     `--name jellyfin \`  
     `--volume /mnt/jellyfin_config:/config \`  
     `--volume /mnt/jellyfin_cache:/cache \`  
-    `--volume /path/to/media:/mnt/library \`  
-    `--publish 8096:8096 \`  
+    `--volume /path/to/media:/media:ro \`  
```

Comment:
I'm a fan of using `:ro` myself, but I think it technically breaks some features. Not sure how I feel about advertising it as the default.

Also - keep the example paths consistent. Either make all `/mnt/whatever` or `/path/to/whatever`.

Reply:
I had changed this due to the recent Emby scare, but you're correct that this conflicts with Jellyfin's ability to delete media from the menu options.  I'll remove the `:ro` tag.

The paths used here were from the original example and follow the existing installation steps.  I will modify them to read as `/path/to/config` and `/path/to/cache` and revise the instructions to be a bit more clear.